docs/multi-echo.rst

```diff
@@ -462,6 +462,19 @@ time point.
     research to better understand how these steps can be applied before denoising.
 
 
+4. Apply susceptibility distortion correction consistently across echoes
+========================================================================
+
+One key feature of susceptibility distortion is that it is primarily a factor of echo spacing, rather than echo time.
+This means that, for most multi-echo sequences, even though dropout will increase with echo time,
+distortion will not (at least not to a noticeable/meaningful extent).
+
+For this reason, if you are applying TOPUP-style (blip-up/blip-down) "field maps",
+we recommend using your first echo time, as this will exhibit the least dropout.
+If your first echo time is very short, and exhibits poor gray/white contrast, then a later echo time may be preferable.
+In any case, you should calculate the spatial transform from just one of your echoes and apply it across all of them.
+
```

Comment:
Lots of repetition in these two files. Not sure if we should add the warning about nonlinear transforms here as well.

Reply:
I think this is ok, given that the nonlinear issues are relatively rare (I think and also hope). It also seems that the fix for this would not be on the users to generate special workflows, but instead on AFNI/fmriprep to run or save things at specific stages.